usaspending_api/api_contracts/contracts/v2/idvs/amounts/award_id.md

```diff
@@ -22,11 +22,18 @@ Get aggregated award counts and funding amounts for IDV contracts
         * `child_award_total_obligation`: 363410.59 (required, number)
         * `child_award_base_and_all_options_value`: 297285.59 (required, number)
         * `child_award_base_exercised_options_val`: 297285.59 (required, number)
+        * `child_total_account_outlay`: 0 (required, number)
+        * `child_total_account_obligation`: 0 (required, number)
+        * `child_account_outlays_by_defc`: [] (required, array[DEFCAmount])
+        * `child_account_obligations_by_defc`: [] (required, array[DEFCAmount])
```

Comment:
These arrays (and the ones below) need `fixed-type` else the type isn't enforced.

Reply:
Now you're just messin' with me aren't you...